packages/ezflow_extension/ezextension/ezflow/modules/ezflow/zone.php

```diff
@@ -30,6 +30,9 @@
 $zoneID = $Params['ZoneID'];
 
 $contentObjectAttribute = eZContentObjectAttribute::fetch( $contentObjectAttributeID, $version );
+if ( !$contentObjectAttribute )
```

Comment:
I think our CS requires curly brackets even for oneliner blocks, though maybe not for legacy?